specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosProtectionPlan.json

```diff
@@ -180,6 +180,58 @@
             "$ref": "./examples/DdosProtectionPlanCreate.json"
           }
         }
+      },
+      "patch": {
+        "tags": [
+          "DdosProtectionPlans"
+        ],
+        "operationId": "DdosProtectionPlans_Update",
```

Comment:
Most Network resources use `_UpdateTags` for PATCH instead of just `_Update`. Let's rename this operationId to keep consistency